concept/ConceptManager.java

```diff
@@ -248,7 +247,7 @@ static void writeAttributeType(StringBuilder builder, AttributeType attributeTyp
             writeOwns(builder, attributeType);
             writePlays(builder, attributeType);
             builder.append(StringBuilders.SEMICOLON_NEWLINE_X2);
-            attributeType.getSubtypesExplicit()
+            attributeType.getSubtypesExplicit().stream()
```

Comment:
these have stayed as streams because we use `sorted()`, we could re-implement a bit later if we wanted. I was aiming to remove the use of `Stream` from method signatures